go/teams/transactions.go

```diff
@@ -0,0 +1,501 @@
+// Copyright 2018 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package teams
+
+import (
+	"errors"
+	"fmt"
+
+	"golang.org/x/net/context"
+
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/keybase1"
+)
+
+type AddMemberTx struct {
+	team     *Team
+	payloads []interface{} // *SCTeamInvites or *keybase1.TeamChangeReq
```

Comment:
Order of these is important so I think `interface{}` and casting still results in simpler code rather than storing two lists and ordering information.